storage/local/index.ts

```diff
@@ -126,6 +126,7 @@ export type SimpleAccount = {
   chainId: number
   address: HexString
   ownerPrivateKey: HexString
+  tokens: Token[]
```

Comment:
Maybe we can remove tokens in config at this moment. It needs more discussion about what must include when backup & import account.

Reply:
If I remove the tokens: Token[], it [(1)](https://github.com/pilagod/waallet-extension/pull/105/files#diff-cc45e3d7aea057eb5b02ad4b0f29a964d020b88a545124dd6d96f7baa2571d92R104), [(2)](https://github.com/pilagod/waallet-extension/pull/105/files#diff-cc45e3d7aea057eb5b02ad4b0f29a964d020b88a545124dd6d96f7baa2571d92R224) will cause a 'Property does not exist' error.

Reply:
After checking, the `tokens` are needed for `Storage`, not the `Account` itself (even when the `Account` backup step is in the feature).
I'll remove the `tokens` from `storage/local/index.ts`.